sonic-slave/Dockerfile.j2

```
@@ -363,3 +363,4 @@ RUN echo "DOCKER_OPTS=\"--experimental --storage-driver=vfs\"" >> /etc/default/d
 RUN echo "deb [arch={{ CONFIGURED_ARCH }}] http://archive.debian.org/debian jessie-backports main" >> /etc/apt/sources.list
 RUN apt-get -o Acquire::Check-Valid-Until=false update
 RUN apt-get -y -o Acquire::Check-Valid-Until=false install ca-certificates-java=20161107~bpo8+1 openjdk-8-jdk
+RUN dpkg --clear-avail; apt-get update
```

This might fix the build:

```suggestion
RUN dpkg --clear-avail; apt-get -o Acquire::Check-Valid-Until=false update
```
One question, though: would it not work to replace line 364 with this line?

---

Thanks Joe!

I should have tested with master branch before creating the PR. There is enough difference between 201811 and master causing me unable to cherry-pick directly.

---

Sorry. Looks like I missed something again in the second try.

---

Strange. I wouldn't think you'd need to make the same change for the Stretch container...

---

My understanding is that the change is to cleanup the environment.

Master might not need Jessie change and 201811 might not need stretch change. But change both anyways to keep it consistent.

---

Clarified offline and you have already made the change, but for posterity, I meant that the `-o Acquire::Check-Valid-Until=false` option should not be needed when building the Stretch container because there are currently no expired repos (whereas the "jessie-backports" repo is no longer maintained and they are no longer updating its "Valid-Until" date.